common/source/docs/common-iflight-beastf7AIO.rst

```diff
@@ -0,0 +1,107 @@
+
+
+-  **Sensors**
+
+   -  InvenSense MPU6000 IMU (accel, gyro) 
+   -  BMP280 barometer
+   -  Voltage & 100A Current sensor
+
+
+-  **Power**
+
+   -  7.4V ~ 25V DC input power
+   -  5V 2.5A BEC for peripherals
+
+
+-  **Interfaces**
+
+   -  5x UARTS
+   -  5x PWM outputs, first 4 are internally connected to 4in1 40A BLHeli32 ESC, the fifth for LED
+   -  I2C port for external compass, airspeed sensor, etc.
+   -  USB port
+   -  8MB on board Flash for logging (set :ref:`LOG_BACKEND_TYPE<LOG_BACKEND_TYPE>` =4)
+   -  Camera input/ VTX output
+   -  Built-in OSD
+
+
+-  **Size and Dimensions**
+
+   - 32.5mm x 32.5mm (25.5 x 25.5mm mount pattern)
+   - 8.5g
+
+
+.. warning:: This board has no compass and no way to connect an external compass. Applications requiring accurate heading information must rely on ArduPilots :ref:`common-compassless` operation.
+
+Where to Buy
+============
+
+`IFlight <https://shop.iflight-rc.com/index.php?route=product/product&product_id=1377>`__
+
+
+Default UART order
+==================
+
+he UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+- SERIAL0 = console = USB
+- SERIAL1 = Telemetry1 = USART1 (Alternate RC input/output)
+- SERIAL2 = Telemetry2 = USART2 (DJI)
+- SERIAL3 = RC Input = USART3 
+- SERIAL4 = GPS = USART4
+- SERIAL7 = USER = UART7 
+
+UART1 supports RX and TX DMA. UART2, UART3 and UART4 support RX DMA. UART7 supports TX DMA
+
+RC Input
+========
+
+RC input is configured by default via the USART3 RX input. It supports all RC protocols except PPM. FPort and full duplex protocols will need to use instead,both RX1 and TX1.
+
+.. note:: If the receiver is FPort or a full duplex protocol, then the receiver must be tied to the USART1 TX pin and :ref:`SERIAL1_OPTIONS<SERIAL1_OPTIONS>` = 7 (invert TX/RX, half duplex), :ref:`SERIAL1_PROTOCOL<SERIAL1_PROTOCOL>` set to 23, and :ref:`RSSI_TYPE<RSSI_TYPE>` =3.
+
+
+Dshot Capability
+================
+
+All motor 1-4 outputs are bi-directional Dshot and PWM capable. However, mixing Dshot and normal PWM operation for outputs is restricted into groups, ie. enabling Dshot for an output in a group requires that ALL outputs in that group be configured and used as Dshot, rather than PWM outputs. The output groups that must be the same (PWM rate or Dshot, when configured as a normal servo/motor output) are: 1/4, and 2/3.
+
+Neopixel Output
+===============
+
+The LED pin is PWM output 5 and is default setup for use with a NeoPixel 4 led string for notifications.
+
+Battery Monitor
+===============
+
+The board has a built-in voltage and current sensors.
+
+The correct battery monitor parameters are:
+
+-    :ref:`BATT_MONITOR<BATT_MONITOR>` =  4
+-    :ref:`BATT_VOLT_PIN<BATT_VOLT_PIN>` = 12
+-    :ref:`BATT_VOLT_MULT<BATT_VOLT_MULT>` ~ 10.9
+-    :ref:`BATT_CURR_PIN<BATT_CURR_PIN>` = 13
+-    :ref:`BATT_AMP_PERVLT<BATT_AMP_PERVLT>` ~ 28.5
```

Comment:
This was wrong in my original definition it should be 100

Reply:
you should set the log backend to 4 also in the defaults

Reply:
so its a 300A sensor?

Reply:
its specs say 200mv/10a which would be 50 for the amp/volt

Reply:
> you should set the log backend to 4 also in the defaults

Not needed - this is done automatically by specifying flash logging

Reply:
> so its a 300A sensor?

The docs say the "current scale: 200", but this value is what gives me believable results

Reply:
okay, if you have measured that 100 is the correct value...will change in both docs...that implies a 300A current sensor range..which I will change also...